src/Components/AppBar.tsx

```diff
@@ -61,7 +62,9 @@ const AppBar = ({open, toggleDrawer, isPermanent, userName}: IAppBarProps & {use
         <BackdropLoader />
       ) : (
         <MyAppBar
+          // sonarignore:start
```

Comment:
If this is to ignore a type issue.
You can try using `Theme` type. import { Theme } from '@mui/material/styles';